### ISSUE TITLE
How to run the app?

### ISSUE THREAD

**Issue by ZedZipDev** [Vahidalizadeh7070/GrpcService#1]
Hi, thank you for the useful article.
It seems it is what I need. I want to add gRPC interface to one of my WCF (SOAP and RESTful) services.

One question: when I run it in VS2022 it cannot be run because of Swagger is not used in the app.
How to run the app and test it as described in the article?

**Comment by Vahidalizadeh7070** [comment]
Hi there. I hope you are well. As I mentioned, postman could be useful to
send a request using grpc. But in this article, swagger works fine since
the request goes through the REST endpoint. What is the problem in your use
case?

**Comment by ZedZipDev** [comment]
When I run the project in VS 2022  the Swagger does not work, Also, I did not find any Swagger packages, and please check the code here
https://github.com/Vahidalizadeh7070/GrpcService/blob/master/GrpcService/Program.cs

If I am wrong please help me to understand where.
Thank you

**Comment by Vahidalizadeh7070** [comment]
Absolutely. There is no swagger. As I mentioned, I have used Postman. If
you need to add swagger, you can register it in the Program.cs file.

builder.Services.AddSwaggerGen();
// Configure the HTTP request pipeline.

if (app.Environment.IsDevelopment())

{

    app.UseSwagger();

    app.UseSwaggerUI();

}

I examined REST using Swagger and it works fine. However, I do not test the
gRPC endpoint. But I think Swagger supports gRPC.
